R/profile_wrapper.R

```diff
@@ -169,7 +226,12 @@ profile_wrapper <- function(mydir, model_settings) {
 
     profilemodels <- r4ss::SSgetoutput(dirvec = profile_dir, keyvec = num)
     profilesummary <- r4ss::SSsummarize(biglist = profilemodels)
-
+    if(!is.null(model_settings$btarg)){
+      profilesummary$btargs <- model_settings$btarg
+      profilesummary$minbthreshs <- model_settings$minbthresh
```

Comment:
I think that this should be `profilesummary$minbthresh`, i.e., not plural. Although, I don't see that it is actually being used in `profile_plot()` because the thresholds are calculated using `ifelse(btarg == 0.40, 0.25, ifelse(btarg == 0.25, 0.125, -1))`.

Reply:
I accidentally grabbed this out of SSsummarize incorrectly. Fixed